packages/sonarwhal/src/bin/sonarwhal.ts

```diff
@@ -38,12 +39,14 @@ import * as cli from '../lib/cli';
 process.once('uncaughtException', (err) => {
     console.error(err.message);
     console.error(err.stack);
+    trackException(err);
```

Comment:
Can you double check if `trackException` is async in the library? We are exiting right after and I'm not sure the exception will be sent properly.

Reply:
it is not async, maybe we can add a delay or something, what do you think?

Reply:
Is there any recommendation on the documentation? We should probably add a delay before doing the `process.exit` in that case...